base_wamas_ubl/lib/wamas/test_utils.py

```diff
@@ -29,6 +41,62 @@ def testDict2wamas(self):
             output = dict2wamas(ast.literal_eval(str_dict_input), telegram)
             self.assertEqual(output, expected_output)
 
+    def testGetAddressNamePicking(self):
+        infile = file_open(
+            file_path("tests/samples/UBL2WAMAS-SAMPLE_AUSK_AUSP-DESPATCH_ADVICE.xml")
+        ).read()
+        dict_item = Dotty(xmltodict.parse(infile))
+        address_elements = get_address_elements(dict_item)
+        self.assertEqual(
+            address_elements,
+            {
+                "ContactName": "Chester Reed",
+                "PartyName": "YourCompany",
+                "Department": "Department_1",
+                "StreetName": "250 Executive Park Blvd, Suite 3400",
+                "AdditionalStreetName": "AdditionalStreetName_1",
+            },
+        )
+        args = (address_elements,)
+        result = get_Adrs_Name(*args)
+        self.assertEqual(result, "Chester Reed")
```

Comment:
```suggestion
        self.assertEqual(get_Adrs_Name(address_elements), "Chester Reed")
```

Reply:
let's keep things simple
I usually use tests to understand what are the specs for a function

Reply:
same bellow